pkgs/desktops/gnome/apps/gnome-logs/default.nix

```diff
@@ -58,6 +58,23 @@ stdenv.mkDerivation rec {
   postPatch = ''
     chmod +x meson_post_install.py
     patchShebangs meson_post_install.py
+
+    # for some reason upstream does the following instead of using meson builtins
+    # TODO: tell upstream to use meson builtins / remove the duplicate check
+
+    # dependency('pkg-config', version : '>= 0.24')
```

Comment:
Likely a remnant from porting from autotools: https://gitlab.gnome.org/GNOME/gnome-logs/-/blame/e0a8ab855f4ea318f2d2c5dfb1b314ac2702c987/meson.build#L56

Reply:
https://gitlab.gnome.org/GNOME/gnome-logs/-/merge_requests/47

Reply:
The extraneous mkenums check was removed in https://gitlab.gnome.org/GNOME/gnome-logs/-/merge_requests/37

Reply:
pulled both in patches